cves/CVE-2013-2251.yml

```diff
@@ -131,7 +131,7 @@ subsystem:
     Look at the path of the source code files code that were fixed to get
     directory names. Look at comments in the code. Look at the bug reports how
     the bug report was tagged. Examples: "clipboard", "gpu", "ssl", "speech", "renderer"
-  answer:
+  answer: DefaultActionMapper
   name:
```

Comment:
Please fill out the name. Also, why did you place this vulnerability in the DefaultActionMapper subsystem?

Reply:
I set it as DefaultActionMapper as that is the directory structure it is in, as well as the subsystem mentioned in the `https://cwiki.apache.org/confluence/display/WW/S2-016` write-up. If I am mistaken as to what this field is supposed to include, please reply again.

Reply:
Sounds good, just make sure to mention that in the file after answer :)
e.g.

answer: _Due to the file structure and Struts security bulletin, I placed this in the DefaultActionMapper subsystem_
name: _DefaultActionMapper_

Reply:
change made